Tests/PythonTests/test_deck_info.py

```diff
@@ -0,0 +1,66 @@
+"""
+Copyright (c) 2019 Chris Ohk, Youngjoong Kim, SeungHyun Jeon
+
+We are making my contributions/submissions to this project solely in our
+personal capacity and are not conveying any rights to any intellectual
+property of any third parties.
+"""
+
+import pyRosetta
+
+def test_constructors():
+    deck1 = pyRosetta.DeckInfo()
+
+    assert deck1.name() == 'Empty'
+    assert deck1.deck_class() == pyRosetta.CardClass.INVALID
+    assert deck1.num_of_cards() == 0
+
+    deck2 = pyRosetta.DeckInfo('Ice Magician', pyRosetta.CardClass.MAGE)
+    
+    assert deck2.name() == 'Ice Magician'
+    assert deck2.deck_class() == pyRosetta.CardClass.MAGE
+    assert deck2.num_of_cards() == 0
+
+def test_card_control():
+    druid_cards = pyRosetta.Cards.find_card_by_class(pyRosetta.CardClass.DRUID)
+    mage_cards = pyRosetta.Cards.find_card_by_class(pyRosetta.CardClass.MAGE)
+
+    deck = pyRosetta.DeckInfo('Ice Magician', pyRosetta.CardClass.MAGE)
+    deck.show_card_list()
+    assert deck.add_card(mage_cards[0].id, 1) == True
+    assert deck.card(0)[1] == 1
+    assert deck.add_card(mage_cards[0].id, 1) == True
+    assert deck.card(0)[1] == 2
+    assert deck.add_card(mage_cards[0].id, 1) == False
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [comparison to False should be 'if cond is False:' or 'if not cond:'](https://app.codacy.com/app/utilForever/RosettaStone/pullRequest?prid=4593740)